app/assess/templates/assessor_tasklist.html

```diff
@@ -49,7 +49,7 @@ <h2 class="assessment-alert__heading govuk-heading-l govuk-!-margin-top-4" autof
             </div>
         </div>
 
-    {% elif sub_criteria_status_completed and current_user_role != "COMMENTER" and not flag.is_qa_complete %}
+        {% elif flag.flag_type.name=="RESOLVED" and sub_criteria_status_completed and current_user_role != "COMMENTER" and not flag.is_qa_complete %}
```

Comment:
We could probs add a test to check for this condition

Reply:
Added